crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -165,58 +203,111 @@ async fn recv_incoming_admin_msgs<A: InterfaceApi>(api: A, rx_from_iface: Websoc
 /// polling for signals being broadcast from the Cells associated with this
 /// App interface.
 fn spawn_recv_incoming_msgs_and_outgoing_signals<A: InterfaceApi>(
+    task_list: &mut TaskList,
     api: A,
     rx_from_iface: WebsocketReceiver,
     rx_from_cell: broadcast::Receiver<Signal>,
     tx_to_iface: WebsocketSender,
 ) {
     use futures::stream::StreamExt;
 
-    trace!("CONNECTION: {}", rx_from_iface.remote_addr());
+    trace!("CONNECTION: {}", rx_from_iface.peer_addr());
 
     let rx_from_cell = futures::stream::unfold(rx_from_cell, |mut rx_from_cell| async move {
-        if let Ok(item) = rx_from_cell.recv().await {
-            Some((item, rx_from_cell))
-        } else {
-            None
-        }
-    });
-
-    tokio::task::spawn(rx_from_cell.for_each_concurrent(4096, move |signal| {
-        let mut tx_to_iface = tx_to_iface.clone();
-        async move {
-            trace!(msg = "Sending signal!", ?signal);
-            if let Err(err) = async move {
-                let bytes = SerializedBytes::try_from(signal)?;
-                tx_to_iface.signal(bytes).await?;
-                InterfaceResult::Ok(())
-            }
-            .await
-            {
-                error!(?err, "error emitting signal");
+        loop {
+            match rx_from_cell.recv().await {
+                // We missed some signals, but the channel is still open
+                Err(tokio::sync::broadcast::error::RecvError::Lagged(_)) => {
+                    warn!("SignalChannelOverloaded");
+                    continue;
```

Comment:
```suggestion
fn spawn_recv_incoming_msgs_and_outgoing_signals<A: InterfaceApi>(
    task_list: &mut TaskList,
    api: A,
    rx_from_iface: WebsocketReceiver,
    rx_from_cell: broadcast::Receiver<Signal>,
    tx_to_iface: WebsocketSender,
    port: u16,
) {
    use futures::stream::StreamExt;

    trace!("CONNECTION: {}", rx_from_iface.peer_addr());

    let rx_from_cell = futures::stream::unfold(rx_from_cell, |mut rx_from_cell| async move {
        loop {
            match rx_from_cell.recv().await {
                // We missed some signals, but the channel is still open
                Err(tokio::sync::broadcast::error::RecvError::Lagged(dropped)) => {
                    warn!("The websocket receiver on port {port} dropped {dropped} messages. The client is sending messages too fast.");
                    continue;
```

Reply:
Actually I take this back. This is the receiver from the cell, so the port is not how we want to identify this.

Reply:
I think the port is fine since it's the app port, but I rewrote your message to try to make it clear the problem is that holochain is emitting signals too fast:

```rust
warn!("Holochain app port {port} dropped {dropped} signals. The app is emitting signals too fast.");
```